frontend/src/main/scala/bloop/logging/ScribeAdapter.scala

```diff
@@ -5,7 +5,7 @@ import scribe.LogRecord
 trait ScribeAdapter extends scribe.LoggerSupport { self: Logger =>
   override def log[M](record: LogRecord[M]): Unit = {
     import scribe.Level
-    val msg = record.message
+    val msg = record.logOutput.plainText
```

Comment:
Why do we need this change?

Reply:
scribe comes transitively from jsonrpc and it was upgraded.
Looks like previously `recod.message` was a String and now it `LazyMessage[A]`